### PR TITLE
Fix image list for metadata editor

### PR DESCRIPTION
**Pull request by Beacze**
Should be rebased after #863 is merged

### CODE REVIEW

File: Kitodo/src/main/java/org/kitodo/services/file/FileService.java [hunk 1]
```diff
@@ -564,48 +569,77 @@ public URI getProcessBaseUriForExistingProcess(Process process) {
...
+                String[] split = uriString.split(processSpecificPartOfUri);
+                String shortUri = split[1];
+                unmapedURI.add(URI.create(shortUri));
+            }
+            unmapedURI.add(uri);
         }
+        return unmapedURI;
+    }
 
+    private String getProcessSubType(Process process, ProcessSubType processSubType, String folderName) {
```
Comment (Kathrin-Huber) [inline Kitodo/src/main/java/org/kitodo/services/file/FileService.java]: should be "ressourceName"

Comment (stweil) [inline Kitodo/src/main/java/org/kitodo/services/file/FileService.java]: Better: `resourceName`.

Comment (henning-gerhardt) [inline Kitodo/src/main/java/org/kitodo/services/file/FileService.java]: Should be done by a rebase against master as `resourceName` is already used (change was in PR #850).

**Comment by henning-gerhardt** [comment]
Please resolve merge conflicts.

**Comment by henning-gerhardt** [comment]
There is a merge conflict. Please solve this conflict.

**Comment by henning-gerhardt** [comment]
@stweil and @Kathrin-Huber : is everything now OK with this pull request?

**Comment by stweil** [comment]
Thanks, my comment was addressed.

**Pull request merged**